src/main/java/ua/goit/gojava32/kickstarter/controller/ProjectController.java

```diff
@@ -0,0 +1,111 @@
+package ua.goit.gojava32.kickstarter.controller;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import ua.goit.gojava32.kickstarter.model.Category;
+import ua.goit.gojava32.kickstarter.model.Project;
+import ua.goit.gojava32.kickstarter.service.CategoryService;
+import ua.goit.gojava32.kickstarter.service.CommentService;
+import ua.goit.gojava32.kickstarter.service.ProjectService;
+import ua.goit.gojava32.kickstarter.service.UserService;
+
+import java.security.Principal;
+import java.util.List;
+
+@Controller
+public class ProjectController {
+
+  @Autowired
+  ProjectService projectService;
+  @Autowired
+  CommentService commentService;
+  @Autowired
+  CategoryService categoryService;
+  @Autowired
```

Comment:
if vars are annotated put empty line between them so it will be readable. also why these are not private?